src/structures/vroom/raw_route.cpp

```diff
@@ -80,7 +80,7 @@ void RawRoute::update_amounts(const Input& input) {
     _nb_pickups[i] = current_nb_pickups;
     _nb_deliveries[i] = current_nb_deliveries;
   }
-  assert(_pd_loads.back() == input.zero_amount());
```

Comment:
Makes sure everything picked along the route has been delivered somehow. OK for removal.